src/shared/MapJson.ts

```diff
@@ -0,0 +1,26 @@
+import path from "path";
+import { promises as fs } from "fs";
+
+export const readMapJsonFiles = async (latitudes: string[]) => {
```

Comment:
can move this function to shared/Map.ts

Reply:
It cannot be placed in shared/Map.ts because shared/map.ts is used in some front-end components that do not support the fs module.